lib/routes/ieee/author.ts

```diff
@@ -0,0 +1,94 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import path from 'node:path';
+import { art } from '@/utils/render';
+import { getCurrentPath } from '@/utils/helpers';
+
+export const route: Route = {
+    name: 'IEEE Author Articles',
+    maintainers: ['Derekmini'],
+    categories: ['journal'],
+    path: '/author/:aid/:sortType/:count?',
+    parameters: {
+        aid: 'Author ID',
+        sortType: 'Sort Type of papers',
+        count: 'Number of papers to show',
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: true,
+    },
+    example: '/ieee/author/37264968900/newest/20',
+    handler,
+};
+
+async function handler(ctx) {
+    const { aid, sortType, count = 10 } = ctx.req.param();
+    const host = 'https://ieeexplore.ieee.org';
+
+    const author = await ofetch(`${host}/rest/author/${aid}`).then((res) => res[0]);
+    const title = `${author.preferredName} on IEEE Xplore`;
+    const link = `${host}/author/${aid}`;
+    const description = author.bioParagraphs.join('<br/>');
+
+    const response = await ofetch(`${host}/rest/search`, {
+        method: 'POST',
+        body: {
+            rowsPerPage: count,
+            searchWithin: [`"Author Ids": ${aid}`],
+            sortType,
+        },
+    });
+
+    const list = response.records.map((item) => {
+        const $ = load(item.articleTitle);
+        return {
+            title: $.text(),
```

Comment:
`articleTitle` is a plaintext only which does not require cheerio
```suggestion
        return {
            title: item.articleTitle,
```

Reply:
```
    const list = response.records.map((item) => ({
        title: item.articleTitle,
        link: item.htmlLink,
        doi: item.doi,
        authors: 'authors' in item ? item.authors.map((itemAuth) => itemAuth.preferredName).join('; ') : 'Do not have author',
        abstract: 'abstract' in item ? item.abstract : '',
    }));
```